shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -171,6 +174,19 @@ const CodeEditor = createClass({
 		this.updateSize();
 	},
 
+	indent : function () {
+		const cm = this.codeMirror;
+			if (cm.somethingSelected()) {
+				cm.execCommand('indentMore');
+			} else {
+				cm.execCommand('insertSoftTab');
+			}
```

Comment:
ironic indent?

Reply:
I'm not sure what you mean?

Reply:
Hmm .. sorry, my snippet wasn't 100% clear.

Line 178 is indented 2 tab stops, but the sibling statement immediately following (lines 179:183) is indented 3 tab stops.